### DISCUSSION TITLE
no native (video) denoise with gpu-next?

### DISCUSSION THREAD

**Discussion by geextahslex** [mpv-player/mpv#14531] [General]
Hi, which possibilities for video denoise are there with gpu-next? From reading the manual there seem to be none with gpu-next

Thank you for any suggestions :)

**Comment by ghost** [comment]
https://github.com/AN3223/dotfiles/blob/master/.config/mpv/shaders/HQ/nlmeans.glsl

**Comment by geextahslex** [comment]
ffmpeg: atadenoise=s=5

script here: https://github.com/mpv-player/mpv/discussions/14519
